### DISCUSSION TITLE
Welcome San as the newest core dev!

### DISCUSSION THREAD

**Discussion by patrick91** [strawberry-graphql/strawberry#2405] [General]
I've made a spoiler about this in our [last community meeting notes](https://github.com/strawberry-graphql/strawberry/discussions/2404), but I think this news deserves its own discussion 😊

I'm so happy to finally announce that @skilkis joined us as a core dev, he's been working on a quite a few PRs in the last year and made a few proof of concepts for improving the performance of Strawberry. I can't wait to see some of these getting merged at some point next year.

Thank you for all the contributions @skilkis! ❤️

<img src="https://media.giphy.com/media/MXwMYSCpDu2TOzZPRg/giphy.gif" width="300">

**Comment by skilkis** [comment]
Thanks @patrick91! I'm excited to join the team and work together with all of you and the community 🍓
